stdlib/2/__builtin__.pyi

```diff
@@ -482,6 +483,8 @@ class tuple(Sequence[_T_co], Generic[_T_co]):
     def __le__(self, x: Tuple[_T_co, ...]) -> bool: ...
     def __gt__(self, x: Tuple[_T_co, ...]) -> bool: ...
     def __ge__(self, x: Tuple[_T_co, ...]) -> bool: ...
+    def __eq__(self, x: Tuple[_T_co, ...]) -> bool: ...
+    def __ne__(self, x: Tuple[_T_co, ...]) -> bool: ...
```

Comment:
This is also not strictly true -- you can write `(1,) == 0` and it'll just return `False`.

However, there's discussion in https://github.com/python/mypy/issues/1271, so I'm not sure what to do.